ops.md

```diff
@@ -28,7 +28,27 @@ Consistently delivers systems and features that provide significant value to cus
 
 ### Domain Knowledge
 
-Mid-to-high level Linux system administration; proficient with UA operations tools/style and contributes to new or updated tooling solutions; can provide troubleshooting across most parts of the stack; able to solve majority operational issues with the stack without escalation; participate in an on-call rotation.
+Mid-level Linux system administration; proficient with UA operations tools/style; can provide troubleshooting across some parts of the stack; able to solve some operational issues without escalation; knows when to escalate to request assistance;  participates in in-hours on-call rotation.
```

Comment:
I think the line between being on-call at night and not is not exactly the line where the person can single-handedly diagnose and solve any issue. There is in implication, in my opinion, that there will be a period where someone is on call, but may have to call someone else in ops on the phone at 2:00AM because things aren't documented adequately, or are complicated enough to need multiple sets of eyes..